### PR TITLE
Fix Noble, Nightcrwawler and Oborin Syndrome perks

### PR DESCRIPTION
**Pull request by Evankhell561**
## About The Pull Request

1. Noble perk weapons can now be selected as oddity when leveling up.

2. Dark  slowdown is canceled by night vision.

## Changelog
:cl:
tweak: Dark slowdown is canceled by night vision.
fix: Oborin Syndrome perk no longer remains after death.
fix: Noble perk.
/:cl:

### CODE REVIEW

File: code/modules/organs/internal/eyes.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	owner.update_client_colour()
 
 /obj/item/organ/internal/eyes/proc/get_colourmatrix() //Returns a special colour matrix if the eyes are organic and the mob is colourblind, otherwise it uses the current one.
-	if(!(BP_IS_ROBOTIC(src)) && owner.stats.getPerk(PERK_OBORIN_SYNDROME))
+	if(!(BP_IS_ROBOTIC(src)) && owner.stats.getPerk(PERK_OBORIN_SYNDROME) && owner.stat != DEAD)
```
Comment (TorinoFermic) [inline code/modules/organs/internal/eyes.dm:53]: You can write owner.is_dead() instead

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 2]
```diff
@@ -13,10 +13,10 @@
 	if(isturf(loc))
 		var/turf/T = loc
 		if(T.get_lumcount() < 0.6)
-			if(!stats.getPerk(PERK_NIGHTCRAWLER))
-				tally += 0.5
-			else
+			if(stats.getPerk(PERK_NIGHTCRAWLER))
 				tally -= 0.5
+			else if(see_invisible != SEE_INVISIBLE_NOLIGHTING)
+				tally += 0.5
```
Comment (ACCount12) [inline code/modules/mob/living/carbon/human/human_movement.dm:19]: What is the purpose of this?

File: code/modules/organs/internal/eyes.dm [hunk 3]
```diff
@@ -50,7 +50,7 @@
 	owner.update_client_colour()
 
 /obj/item/organ/internal/eyes/proc/get_colourmatrix() //Returns a special colour matrix if the eyes are organic and the mob is colourblind, otherwise it uses the current one.
-	if(!(BP_IS_ROBOTIC(src)) && owner.stats.getPerk(PERK_OBORIN_SYNDROME))
+	if(!(BP_IS_ROBOTIC(src)) && owner.stats.getPerk(PERK_OBORIN_SYNDROME) && owner.is_dead())
```
Comment (ACCount12) [inline code/modules/organs/internal/eyes.dm:53]: Why is this check here? Wouldn't it mean that `PERK_OBORIN_SYNDROME` only affects dead mobs?

File: code/modules/mob/living/carbon/human/human_movement.dm (hunk 2 above)
Comment (Evankhell561) [reply code/modules/mob/living/carbon/human/human_movement.dm:19]: if you don't have night vision you become slow in the dark.

File: code/modules/organs/internal/eyes.dm (hunk 3 above)
Comment (Evankhell561) [reply code/modules/organs/internal/eyes.dm:53]: this was an error that was inadvertently generated in the previous review.
image

**Pull request merged**